specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/common.json

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
Semantic diff to 2022-03-01-preview in main: https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvbWFpbi9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTAzLTAxLXByZXZpZXcvY29tbW9uLmpzb24iLCJ0YXJnZXRVcmwiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvOTkyMDk5ZGZiZDUyZmY1ZWM3MDBhY2Y4OWQ5Zjc4ZDc5YjYxOThjZS9zcGVjaWZpY2F0aW9uL2NvZ25pdGl2ZXNlcnZpY2VzL2RhdGEtcGxhbmUvTGFuZ3VhZ2UvcHJldmlldy8yMDIyLTA1LTAxLXByZXZpZXcvY29tbW9uLmpzb24ifQ==

Reply:
No changes were detected.